ldblink.cpp

```diff
@@ -658,7 +652,7 @@ RegisterFactory(DBLinkFactory);
 RegisterLibrary (
 	"Maurizio Felici",
 	__DATE__,
-	"0.2.0",
+	"0.2.1",
```

Comment:
We will need to create a process for the releases. There is this option in GitHub. So we can have official releases. @roypaulin Maurizio already built some unit tests, is it possible to build the test env and do official releases?

Reply:
They are not unit tests. But yes Maurizio needed to work a little bit on them to make them simpler (with simple data etc...).He planed to do that on another PR but I guess he should finally do that on the current one.